src/components/button/button.tsx

```diff
@@ -0,0 +1,35 @@
+import { MouseEventHandler } from "react";
+
+type buttonProps = {
+    
+    buttonText?: string;
+    onClick?: MouseEventHandler;
+    icon?:string;
+  };
+
+export const ButtonPrimary = ({buttonText="value", onClick}:buttonProps) => {
+    return (
+        <button className="text-lightGray bg-[#121212] px-10 rounded-lg grow-0 h-14  w-80 text-lg font-medium font-text" onClick={onClick}>{buttonText}</button>
```

Comment:
for each button please add ```<button className={`text-lightGray bg-[#121212] px-10 rounded-lg grow-0 h-14  w-80 text-lg font-medium font-text ${className}`}``` This will make the button styles editable in other components

Reply:
Added classname prop of type string to classname property of button element